app/helpers/application_helper/toolbar_chooser.rb

```diff
@@ -386,7 +386,7 @@ def center_toolbar_filename_classic
     to_display = %w[availability_zones cloud_networks cloud_object_store_containers cloud_subnets configured_systems
                     cloud_tenants cloud_volumes ems_clusters flavors floating_ips host_aggregates hosts
                     network_ports network_routers network_services orchestration_stacks resource_pools
-                    security_groups security_policies security_policy_rules storages]
+                    security_groups security_policies security_policy_rules storages storage_resources]
```

Comment:
we shouldn't need empty toolbars if `storage_resource` is not added here

Reply:
removed